empress_gui.py

```diff
@@ -0,0 +1,335 @@
+# empress_gui.py
+
+import tkinter as tk
+from tkinter import messagebox
+from pathlib import Path
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg, NavigationToolbar2Tk
+from matplotlib.backend_bases import MouseEvent, key_press_handler
+import empress
+
+class App:
+
+    def __init__(self, master):
+        self.master = master
+        # configures the master frame 
```

Comment:
I think it might look better to use capital letter to start a block comment.

Reply:
Done:)